client/app/kubernetes/translations/Messages_fr_FR.json

```diff
@@ -41,6 +41,8 @@
   "kube_nodes_add_region": "Durant la bêta, vos nœuds seront tous dans la région GRA5. Nous permettrons ultérieurement de choisir cette région.",
   "kube_nodes_add_success": "Le nœud est en cours d'installation",
   "kube_nodes_add_flavor_error": "Une erreur est survenue lors du chargement des modèles d'instance : {{ message }}",
+  "kube_nodes_add_node_name": "Nom de l'instance",
+  "kube_nodes_add_node_name_validation_error": "Le nom doit commencer et se terminer par des caractères alphanumériques minuscules et peut comporter des caractères alphanumériques minuscules, '.' et '-'",
```

Comment:
```suggestion
  "kube_nodes_add_node_name_validation_error": "Le nom doit commencer et se terminer par des caractères alphanumériques minuscules et peut comporter des caractères alphanumériques minuscules, ainsi que '.' et '-'",
```

Reply:
The text was provided by CX and has been updated now